### ISSUE TITLE
Shipping Charges Doubling Up in Paypal

### ISSUE THREAD

**Issue by robertallenpayne** [nbrightproject/NBrightBuy#23]
Hi David,

I've noticed a problem with how shipping is being calculated.  At first, I set up my shipping to be based on pricing. So for example,
0-120=6.00
120-220=12.00
etc

So when I would order one tee shirt at $ 20.00, the cart would show the total with shipping to be $ 26.00.  But when this got to PayPal, it would show $ 32.00. 

So I changed my shipping to be based on weight.  But I got the same thing.

It's like it is sending the total with shipping to paypal as just the total, and then sending the shipping along also.

Can we get this fixed?  I've got customers who are losing sales due to this issue.

Thanks!

**Comment by leedavi** [comment]
Hi, Robert,

Check your paypal settings, there are options to add shipping in there.  It sounds like the paypal shipping option is adding the extra.

**Comment by robertallenpayne** [comment]
Hi David,

I apologize for the delay in responding.  It took me a while to actually coordinate my schedule with my customers so we could go digging into the PayPal setup.  We did find the setting and were able to turn it off.  It's a small thing, but I appreciate you pointing it out to me.  Solved the problem for sure.

For other folks who might have this same issue - once you log into your PERSONAL PayPal account (Not sure how this will work for Business accounts), you will navigate to Shipping Calculations.  You will need to set up at least one Shipping Calculation and set it to add nothing.  I didn't make detailed notes when I was working with my customer, but it was pretty easy to figure out.
